tensorflow/lite/experimental/kernels/BUILD

```diff
@@ -35,10 +35,21 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "gru_cell",
+    srcs = ["gru_cell.cc"],
+    hdrs = ["gru_cell.h"],
+    deps = [
+        "//tensorflow/lite/kernels/internal:optimized",
+        "//third_party/eigen3",
+    ],
+)
+
 cc_library(
     name = "experimental_ops",
     srcs = [
         "ctc_beam_search_decoder.cc",
+        "unidirectional_sequence_gru.cc",
```

Comment:
I think it would be better for users to choose what ops to include for experimental ops.

so you can have 
target: ctc_beam_search_decoder_op, unidirectional_sequence_gru_op, etc.

no need to bundle them together

Reply:
Done